README.md

```diff
@@ -28,6 +28,8 @@ module "pubsub" {
       x-goog-version             = "v1beta1"                                            // optional
       oidc_service_account_email = "sa@example.com"                                     // optional
       audience                   = "example"                                            // optional
+      no_wrapper                 = false                                                // optional
```

Comment:
If possible, please add any new variables an example so they are exercised in the Integration CI: https://github.com/terraform-google-modules/terraform-google-pubsub/tree/master/examples